src/goto-checker/symex_bmc_incremental_one_loop.cpp

```diff
@@ -85,6 +88,14 @@ bool symex_bmc_incremental_one_loopt::should_stop_unwind(
     abort_unwind_decision.is_known(), "unwind decision should be taken by now");
   bool abort = abort_unwind_decision.is_true();
 
+
+  if(output_ui == ui_message_handlert::uit::XML_UI)
+  {
+    xmlt xml("current-unwinding");
+    xml.data = std::to_string(unwind);
+    log.statistics() << xml;
+  }
```

Comment:
Could we also output a `current-unwinding` item for `PLAIN` and `JSON_UI`? So the same information is output independent of the  ui.

Reply:
Done.